homeassistant/components/heos/media_player.py

```diff
@@ -122,12 +136,66 @@ async def async_added_to_hass(self):
                 SIGNAL_HEOS_UPDATED, self._heos_updated
             )
         )
+        self._group = await self.async_get_group()
 
     @log_command_error("clear playlist")
     async def async_clear_playlist(self):
         """Clear players playlist."""
         await self._player.clear_queue()
 
+    async def async_get_group(self):
+        """Build a dictionary with HEOS group membership information."""
+        group_membership = []
+        if not self.entity_id:
```

Comment:
This condition should never occur when only called from within `async_update()`

Reply:
OK, I removed the check.